### DISCUSSION TITLE
I am facing problem while using quasar sessionstorage

### DISCUSSION THREAD

**Discussion by narendraparihar** [quasarframework/quasar#13635] [CLI - PWA mode] [answered]
I have gone through the complete sessionstorage guideline on quasar's main website. I am getting this error whenever i am trying to use quasar session storage

"Uncaught TypeError: Cannot read properties of undefined (reading 'sessionStorage')"

I have imported it in file in which i am using it and the root file as well.

**Comment by MilosPaunovic** [comment]
Please share some code segments with us, how you're importing it, how you're using it...

**Reply by narendraparihar** [reply]
//quasar.config.js file 

framework: {
    plugins: [
      'LocalStorage',
      'SessionStorage'
    ]
  }

//app.vue
import {Quasar, LocalStorage, SessionStorage } from 'quasar'

//file in which I want this feature
import { useQuasar } from 'quasar'
       const $q = useQuasar()   
      $q.sessionStorage.set('ak_user', 'narendra')

**Reply by metalsadman** [reply]
it should work.

**Comment by pdanpdan** [comment]
Did you add the plugin in quasar.conf?

**Reply by narendraparihar** [reply]
yes

**Comment by suleimanas** [comment, marked as answer]
try it with an empty template something like that:
```
<template>
  <q-page>
  </q-page>
</template>

<script setup>
  import { useQuasar } from 'quasar'
  const $q = useQuasar()
  $q.sessionStorage.set('ak_user', 'narendra')
</script>
```
does it work?

**Reply by narendraparihar** [reply]
I forgot to add this keyword with $q. Now It is working completely fine. Thank you everyone
